### ISSUE TITLE
The headers are not correct anymore

### ISSUE THREAD

**Issue by amtoine** [goatfiles/dotfiles#2]
Since i changed my username from `a2n-s` to `amtoine`, which i'm more happy with, the headers of most of the config files are not correct anymore!

**Needed changes**
i'll take the example of the `amtoine-battery` script in my [scripts](https://github.com/amtoine/scripts), see [#18](https://github.com/amtoine/scripts/issues/18), which has the exact same problem.
the changes that should be applied:
```diff
diff --git a/scripts/amtoine-battery b/scripts/amtoine-battery
index 4642f77..beb2849 100755
--- a/scripts/amtoine-battery
+++ b/scripts/amtoine-battery
@@ -1,13 +1,13 @@
 #!/usr/bin/env bash
-#           ___
-#      __ _|_  )_ _ ___ ___   personal page: https://amtoine.github.io/
-#     / _` |/ /| ' \___(_-<   github   page: https://github.com/amtoine
-#     \__,_/___|_||_|  /__/   my   dotfiles: https://github.com/amtoine/dotfiles
-#           ___                _          _   _
-#      __ _|_  )_ _ ___ ______| |__  __ _| |_| |_ ___ _ _ _  _
-#     / _` |/ /| ' \___(_-<___| '_ \/ _` |  _|  _/ -_) '_| || |
-#     \__,_/___|_||_|  /__/   |_.__/\__,_|\__|\__\___|_|  \_, |
-#                                                         |__/
+#                _       _
+#      __ _ _ __| |_ ___(_)_ _  ___    personal page: https://amtoine.github.io/
+#     / _` | '  \  _/ _ \ | ' \/ -_)   github   page: https://github.com/amtoine
+#     \__,_|_|_|_\__\___/_|_||_\___|   my   dotfiles: https://github.com/amtoine/dotfiles
+#                _       _                  _          _   _
+#      __ _ _ __| |_ ___(_)_ _  ___   ___  | |__  __ _| |_| |_ ___ _ _ _  _
+#     / _` | '  \  _/ _ \ | ' \/ -_) |___| | '_ \/ _` |  _|  _/ -_) '_| || |
+#     \__,_|_|_|_\__\___/_|_||_\___|       |_.__/\__,_|\__|\__\___|_|  \_, |
+#                                                                      |__/
 #
 # Description:  a script to check the state of the battery and throw appropriate notifications.
 #   run `sudo systemctl enable cronie` to activate the `cron` daemon if not already done, then
```

**Comment by amtoine** [comment]
i've created this issue a while back when i changed my name from the old "a2n-s" to my new "amtoine" :thinking: 

what should we do from now on @atxr? :yum:

**Comment by amtoine** [comment]
i've tried to spot the files with bad headers, and here is the list, so far, of files whose headers will have to be addressed:
- `.bash_aliases`
- `.bash_logout`
- `.bash_profile`
- `.bashrc`
- `.config/X11/xinitrc`
- `.config/alacritty/alacritty.yml`
- `.config/bspwm/bspwmrc`
- `.config/bspwm/scripts/autostart.sh`
- `.config/bspwm/sxhkd/sxhkdrc`
- `.config/dmenufm/dmenufm.conf`
- `.config/dmscripts/config`
- `.config/git/config`
- `.config/kitty/kitty.conf`
- `.config/lazygit/config.yml`
- `.config/lf/lfrc`
- `.config/lf/scripts/cleaner.sh`
- `.config/lf/scripts/previewer.sh`
- `.config/mpd/mpd.conf`
- `.config/mpv/input.conf`
- `.config/mpv/mplayer-input.conf`
- `.config/mpv/mpv.conf`
- `.config/mpv/restore-old-bindings.conf`
- `.config/ncmpcpp/config`
- `.config/neofetch/config.conf`
- `.config/neofetch/neofetchrc`
- `.config/qtile/bar.py`
- `.config/qtile/config.py`
- `.config/qtile/groups.py`
- `.config/qtile/keys.py`
- `.config/qtile/layouts.py`
- `.config/qtile/mouse.py`
- `.config/qtile/popups.py`
- `.config/qtile/screens.py`
- `.config/qtile/scripts/autostart.sh`
- `.config/qtile/scripts/bar.sh`
- `.config/qtile/scripts/km.sh`
- `.config/qtile/style.py`
- `.config/qtile/utils/utils.py`
- `.config/qtile/widgets.py`
- `.config/spectrwm/spectrwm.conf`
- `.config/spectrwm/spectrwm_fr.conf`
- `.config/spectrwm/spectrwm_us.conf`
- `.config/starship.toml`
- `.config/surf/css/archlinux.css`
- `.config/surf/css/default.css`
- `.config/surf/css/github.css`
- `.config/surf/css/homepage.css`
- `.config/surf/css/kitty.css`
- `.config/surf/css/qtile.css`
- `.config/surf/css/sdd.css`
- `.config/surf/css/stackoverflow.css`
- `.config/surf/css/suckless.css`
- `.config/surf/css/tecmint.css`
- `.config/surf/css/wikipedia.css`
- `.config/surf/css/youtube.css`
- `.config/surf/html/homepage.html`
- `.config/surf/scripts/add.bm.sh`
- `.config/surf/scripts/dmenu.linkselect.sh`
- `.config/surf/scripts/dmenu.mpv.sh`
- `.config/surf/scripts/dmenu.setprop.sh`
- `.config/surf/scripts/dmenu.uri.sh`
- `.config/surf/scripts/edit.bookmarks.sh`
- `.config/surf/scripts/edit.screen.sh`
- `.config/surf/scripts/edit.url.sh`
- `.config/surf/scripts/link_hints.js`
- `.config/surf/scripts/open.help.sh`
- `.config/sxhkd/sxhkdrc`
- `.config/tig/config`
- `.config/tmux/tmux.conf`
- `.config/vifm/vifmrc`
- `.config/www/css/homepage.css`
- `.config/www/html/homepage.html`
- `.profile`

**Comment by atxr** [comment]
I think replacing every occurrence with `goat` for example can be better
It will allow us to make a separation between our personal config and the global goat config

**Comment by amtoine** [comment]
> I think replacing every occurrence with `goat` for example can be better It will allow us to make a separation between our personal config and the global goat config
>
yup definitely :+1: 

but do you mean adding a new header to all the files?
with what content exactly?

**Comment by atxr** [comment]
Oh no I don't think it's necessary, that's a lot of work for nothing.
Just replacing the existing occurrences of `amointe`, `ants`...

**Comment by amtoine** [comment]
> Oh no I don't think it's necessary, that's a lot of work for nothing. Just replacing the existing occurrences of `amointe`, `ants`...
>
mm :thinking: 
the thing is that the ASCII banners themselves are not correct anymore... :confused:
so i think they should be replaced anyways
 
i've already tried to remove all the headers, and that's something totally feasible :+1: 
then the questions would be:
- **do we add another banner in replacement?**
- **if yes, what do we add?**

you know, something simple like
```
                _    __ _ _
 __ _ ___  __ _| |_ / _(_) |___ ___  GitHub: https://github.com/goatfiles
/ _` / _ \/ _` |  _|  _| | / -_|_-<  website: https://goatfiles.github.io
\__, \___/\__,_|\__|_| |_|_\___/__/  LICENSE: https://github.com/goatfiles/dotfiles/blob/main/LICENSE
|___/
            maintainers: 
                https://github.com/amtoine
                https://github.com/atxr 
```
you know, something nice and simple, easy to find, easy to edit, easy to replace and uniform accross all the dotfiles :thinking: 

**in the end, i think all the old banners should go away, replaced or not** :wink:

**Comment by amtoine** [comment]
fixed in #22
